src/ElmSharp/ElmSharp/EvasObject.cs

```diff
@@ -1100,20 +1119,24 @@ public void Unrealize()
 
                 Interop.Evas.evas_object_del(toBeDeleted);
                 Parent = null;
+                _objects.Remove(toBeDeleted);
```

Comment:
I think I should check the Key. Otherwise, it seems that Handle is already Intptr.Zero when it is the second remove.

Reply:
It's not IntPtr.Zero on handle is removed on native side.
although Dictionary.Remove method is does not fail if toBeDeleted is IntPtr.Zero.